dev_guide/builds.adoc

```diff
@@ -1184,6 +1190,80 @@ spec:
 ----
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+Build hooks can be used as a part of the build flow to run tests inside a temporary container and validate your image before it is being pushed to the registry. The command is executed with the current working directory set to the image's `WORKDIR`.
+The hook will fail if the script or command returns a non-zero exit code or if there's any other error related to starting the temporary container. When the hook fails it will mark the build as failed.
+====
+----
+spec:
+  postCommit: <1>
+    command:
```

Comment:
- [ ] According to the description below, the `<1>` marker should be in this line, and not in the line above.
- [ ] JSON is valid YAML, so I think here it would be more clear to write `command: ["/bin/sh", "-c"]` instead of using multiple lines.